.github/pull_request_template.md

```diff
@@ -1,58 +1,38 @@
 
-<!--
-Thank you for opening a pull request!  Here are some tips on creating
-a well formatted contribution.
-
-Please give your pull request a title like "[component]: [short description]"
-
-This is the format for commit messages:
 
-"""
-[component]: [short description]
+<!--
+  - Please give your pull request a title like
 
-[A longer multiline description]
+      [component]: [short description]
 
-Fixes: [ticket URL on tracker.ceph.com, create one if necessary]
-Signed-off-by: [Your Name] <[your email]>
-"""
+  - Please use this format for each git commit message:
 
-The Signed-off-by line is important, and it is your certification that
-your contributions satisfy the Developers Certificate or Origin.  For
-more detail, see SubmittingPatches.rst.
+      [component]: [short description]
 
-The component is the short name of a major daemon or subsystem,
-something like "mon", "osd", "mds", "rbd, "rgw", etc. For ceph-mgr modules,
-give the component as "mgr/<module name>" rather than a path into pybind.
+      [A longer multiline description]
 
-For more examples, simply use "git log" and look at some historical commits.
+      Fixes: [ticket URL on tracker.ceph.com, create one if necessary]
+      Signed-off-by: [Your Name] <[your email]>
 
-This was just a quick overview.  More information for contributors is available here:
-https://raw.githubusercontent.com/ceph/ceph/master/SubmittingPatches.rst
+    For examples, use "git log".
 
+  - The Signed-off-by line in every git commit is important; see SubmittingPatches.rst.
 -->
+
 ## Checklist
-- [ ] References tracker ticket
-- [ ] Updates documentation if necessary
-- [ ] Includes tests for new functionality or reproducer for bug
-
----
-
-<details>
-<summary>Show available Jenkins commands</summary>
-
-- `jenkins retest this please`
```

Comment:
is this list in a doc somewhere else already? useful for reference

Reply:
I installed the ceph chrome extension.  But maybe we should keep this.

Reply:
yes please, I'm using this list sometimes

Reply:
That list should be up to date (e.g.: it contains `jenkins test dashboard cephadm`). I find that is especially useful for for newcomers/sporadic contributors.

The Chrome extension from @tspmelo ? I think the commands there are also outdated (I only use it for filling the PR reviewers).

Reply:
please, i've been using it on daily basis.

Reply:
re-added

Reply:
nit: The formatting seems off in https://github.com/ceph/ceph/blob/71135e2828ce2f60819ed0c4cd565c291758f675/.github/pull_request_template.md, not sure if it is because of removing this `-`